src/tv2_afvd_showstyle/config-manifests.ts

```diff
@@ -262,6 +262,46 @@ export const schemaConfigManifest: ConfigManifestEntry[] = [
 	}
 ]
 
+export const gfxDesignTemplates: ConfigManifestEntry[] = [
+	{
+		id: 'GFXDesignTemplates',
+		name: 'GFX Design Templates',
+		description: '',
+		type: ConfigManifestEntryType.TABLE,
+		required: true,
+		defaultVal: [],
+		columns: [
+			{
+				id: 'INewsName',
+				name: 'iNews Name (*)',
+				description: 'The name after the code',
```

Comment:
`The code` is quite vague. Are you referring to the code base? ;)

Reply:
Was a leftover from copying the GFXTemplate config, will fix in next commit.

Reply:
Changed in [73a488c](https://github.com/tv2/sofie-blueprints-inews/pull/149/commits/73a488c30f009df70d38e047371d4b47fdf749ec)